### PR TITLE
feat: avatar component changes and stories

### PR DESCRIPTION
**Pull request by vozbrann**

### CODE REVIEW

File: src/components/avatar/style.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ import { PUITheme } from 'utils/types';
 export const StyledAvatar = styled.div<{
   theme: PUITheme;
 }>`
-  background-color: ${(props: any): string => props.theme.colors.base200};
+  background-color: ${(props): string => props.theme.colors.base200};
   border-radius: 50%;
-  border: 1px solid ${(props: any): string => props.theme.colors.base300};
+  border: 1px solid ${(props): string => props.theme.colors.base300};
```
Comment (vlacher12) [inline src/components/avatar/style.ts:9]: just take theme from props
`  background-color: ${({ theme }): string => theme.colors.base200};`

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/avatar/index.tsx [hunk 2]
```diff
@@ -1,63 +1,37 @@
...
 }
 
-export const Avatar: React.FC<AvatarProps> = ({ className, src, size = 54, alt, children, imgProps }: AvatarProps) => {
+export const Avatar: React.FC<AvatarProps> = ({
+  className,
+  size = 54,
+  children,
+  imgProps,
+  src,
+  fontSize,
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:21]: can we still support backward compatibility for font sizes?

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/avatar/index.tsx (hunk 2 above)
Comment (vozbrann) [reply src/components/avatar/index.tsx:21]: We removed the `getFontSize` function, so I don't think so

Comment (vlacher12) [reply src/components/avatar/index.tsx:21]: why did we remove it?

Comment (vozbrann) [reply src/components/avatar/index.tsx]: fixed

File: src/components/avatar/index.tsx [hunk 3]
```diff
@@ -1,63 +1,48 @@
...
   children?: React.ReactNode | string;
   className?: string;
   imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
   size?: number;
   src?: string;
 }
 
-export const Avatar: React.FC<AvatarProps> = ({ className, src, size = 54, alt, children, imgProps }: AvatarProps) => {
+export const Avatar: React.FC<AvatarProps> = ({ className, size = 54, children, imgProps, src }: AvatarProps) => {
   const theme = useTheme();
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:32]: destructure pls
`const { typography, colors } = useTheme();`

**Review by vlacher12 — CHANGES_REQUESTED** [review]

**Review by vlacher12 — APPROVED** [review]

**Review by lazzystep — APPROVED** [review]

**Pull request merged**